Kubernetes/HOWTO-on-prem.md

```diff
@@ -1,94 +1,67 @@
+  - The service subnet is assumed to be 11.0.0.0/16 and is hardcoded throughout the scripts. Changes to this value are *not covered.*
 
+The machines can be VMs (which is assumed throughout the guide) or bare-metal hosts. Both should work identically. 
 
 **Note**: The guide will assume a local working directory of `~/kube` for the Kubernetes setup. If you choose a different directory, just replace any references to that path.
 
-## Building the Binaries ##
-We will begin with a Linux master node on a recent Ubuntu-like distro and a Windows worker node on RS3+; all of these are assumed to be VMs.
+**Note**: As of this writing, the latest 1.8 version of Kubernetes was 1.8.2. You can check this [here](https://github.com/kubernetes/kubernetes/releases) and update all references to `1.8.x` accordingly.
 
-First, install all of the pre-requisites on the master:
 
-    $ sudo apt-get install curl git build-essential docker.io conntrack
+## Preparing the Master ##
+First, let's install all of the pre-requisites:
 
-### Building Kubernetes ###
-We will need to build the `kubelet` and `kubeproxy` binaries for Windows from scratch by _cross-compiling from Linux_. There are multiple ways to do this:
+    $ sudo apt-get install curl git build-essential docker.io conntrack
 
-  - Build them [locally](#build-locally).
-  - Generate the binaries using [Vagrant](#build-with-vagrant).
-  - You can leverage the [standard containerized build scripts](https://github.com/kubernetes/kubernetes/tree/master/build#key-scripts) in the Kubernetes project. For this, follow the steps for [building locally](#build-locally) up to the `make` steps, then use the linked instructions.
+We can pull most of the binaries we need from the official Kubernetes repository, but will need to cross-compile one of the Windows binaries ourselves because it isn't "officially released."
 
-**Note**: If you run into "permission denied" errors, these can be avoided by building the Linux kubelet first, per the note in [acs-engine](https://github.com/Azure/acs-engine/blob/master/scripts/build-windows-k8s.sh#L176):
+There is a collection of scripts in [this repository](https://github.com/Microsoft/SDN/tree/master/Kubernetes/linux), which will help us with the setup process. Check them all out to `~/kube/` and make the scripts executable. This entire directory will be getting mounted for a lot of the docker containers in future steps, so keep its structure the same as outlined in the guide.
 
-> Due to what appears to be a bug in the Kubernetes Windows build system, one has to first build a Linux binary to generate `_output/bin/deepcopy-gen`. Building to Windows w/o doing this will generate an empty `deepcopy-gen`.
+    $ mkdir ~/kube
+    $ git clone https://github.com/Microsoft/SDN /tmp/k8s 
+    $ mv -R /tmp/k8s/Kubernetes/linux/ ~/kube/

```

Comment:
We need to add a command to do this for windows scripts as well, correct? I think the section below on setting up windows nodes expects that kube binaries *and scripts* from SDN repo are in ~/kube-win/ dir.